server/src/assignment/assignment.service.ts

```diff
@@ -214,14 +214,15 @@ function generateAssignmentsQueryObject(contact) {
     // filters by projects with associations having some connection to contactid
     // OR infers an association through recodedCbFullName (for prefiled)
     $filter: `
-      (dcp_dcp_project_dcp_communityboarddisposition_project/any
+      dcp_visibility eq 717170003
```

Comment:
We should add a comment as to what 717170003 means (what the human readable label is)

Reply:
Ditto, we'll want to start adding these lookups so we can label stuff:

```javascript
  dcp_visibility eq ${ASSIGNMENT_VISIBILITY.PUBLIC}
```

Or something